test/eval_tests/array.js

```diff
@@ -0,0 +1,10 @@
+'use superstrict';
+
+try {
```

Comment:
This test is bad. There is nothing wrong within the for-cycle, so the 'ok' should be put to the `try` clause, not `catch`. If this test passes, there is probably something wrong in the code.